specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/subscriptions.json

```diff
@@ -848,6 +1052,22 @@
             "name": "ProvisioningState",
             "modelAsString": true
           }
+        },
+        "redeemUrl": {
+          "description": "Redeem url.",
+          "type": "string"
+        },
+        "redeemState": {
```

Comment:
redeemState should be a readonly property

Reply:
Updated